src/chat.rs

```diff
@@ -1907,6 +1908,54 @@ pub fn forward_msgs(context: &Context, msg_ids: &[MsgId], chat_id: u32) -> Resul
     Ok(())
 }
 
+pub fn get_info_json(context: &Context, chat_id: u32) -> Result<String, Error> {
+    let chat = Chat::load_from_db(context, chat_id).unwrap();
+
+    // ToDo:
+    // - [x] id
+    // - [x] type
+    // - [x] name
+    // - [x] archived
+    // - [x] color
+    // - [x] profileImage
+    // - [x] subtitle
+    // - [x] draft,
+    // - [ ] deaddrop,
+    // - [ ] summary,
+    // - [ ] lastUpdated,
+    // - [ ] freshMessageCounter,
+    // - [ ] email
```

Comment:
until those are done i guess it's a draft PR?

Reply:
Depends. To have something in we can merge it and implement those things in a second pr, but implementing the whole thing would be better IMO.

Reply:
ok.